lib/main.dart

```diff
@@ -22,10 +27,21 @@ class ProximaApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final Brightness brightness = MediaQuery.platformBrightnessOf(context);
+
     return MaterialApp(
       title: "Proxima",
       onGenerateRoute: generateRoute,
       initialRoute: Routes.login.name,
+      theme: ThemeData(
+        useMaterial3: true,
+        // Define the default brightness and colors.
+        colorScheme: ColorScheme.fromSeed(
+          seedColor: Colors.indigo,
+          brightness: brightness,
+        ),
+        fontFamily: GoogleFonts.poppins().fontFamily,
+      ),
     );
```

Comment:
Very good point. Setting the theme/font at the top level will ensure that the whole app looks the same. We will need to be carefull in the future not to use custom colors in other widgets.